### PR TITLE
Added Python language assignments.

### LINKED ISSUE #4: Need to add Python language assignments.
For student's practice need to add Python language assignments.

### PR DESCRIPTION
**Pull request by amolkahat**
Added Python language assignments.
Closes #4

### CODE REVIEW

**Comment by amolkahat** [comment]
@akoserwal Please merge the request.

File: languages/python/Assignments/assignment1.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   * * *
+  * * * *
+ * * * * *
+  * * * *
+   * * *
+    * *
+     *
+```
+
+3. WAP for no of occurrences of the characters in the string. Print it’s count and index.
```
Comment (brgnepal) [inline languages/python/Assignments/assignment1.md]: Please replace `WAP` with its full form.

Comment (amolkahat) [reply languages/python/Assignments/assignment1.md]: @budhrg WAP is ok. People will understand it. Just merge this request.

Comment (brgnepal) [reply languages/python/Assignments/assignment1.md]: The majority of the people will not I believe. 

What do you think?
@akoserwal @Jaanki @rahulbajaj0509

**Comment by rabajaj0509** [comment]
Thanks @amolkahat

**Pull request merged**